lua/graph.lua

```diff
@@ -300,7 +300,8 @@ cycleway = {
 ["share_busway"] = "true",
 ["allowed"] = "true",
 ["private"] = "true",
-["cyclestreet"] = "true"
+["cyclestreet"] = "true",
+["crossing"] = "true"
 }
```

Comment:
where is this used? Does this enable cycling access?

Reply:
used for cycleway=crossing.  yes enable it.  see screen shoot of updated route that failed before.